api/development.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine3.15
+FROM node:20.10.0-alpine3.19
 
```

Comment:
It might make sense to add the npm version here as well?
e.g.
`RUN npm install -g npm@10.8.2`

Or to run a shared base image.